### PR TITLE
chore(content-switcher): update styles from audit

### LINKED ISSUE #1956: [Audit] Content Switcher
**States:**
- default
- Content Switcher
- Content Switcher with Icon

🖼 **Sketch file:** https://ibm.ent.box.com/file/336719415945

content switcher - white theme

### PR DESCRIPTION
**Pull request by aledavila**
Closes #1956 

#### Changelog

**Changed**

- border radius
- padding
- align text left
- focus to align with border radius

### CODE REVIEW

File: src/components/content-switcher/_content-switcher.scss [hunk 1]
```diff
@@ -128,7 +128,8 @@
     background-color: $ui-01;
     display: flex;
     align-items: center;
-    padding: $spacing-xs $spacing-xl;
+    height: 40px;
```
Comment (jnm2377) [inline src/components/content-switcher/_content-switcher.scss]: Can you use `rem()` here?

File: src/components/content-switcher/_content-switcher.scss [hunk 2]
```diff
@@ -162,6 +163,16 @@
     }
   }
 
+  .#{$prefix}--content-switcher-btn:first-child {
+    border-top-left-radius: 4px;
+    border-bottom-left-radius: 4px;
```
Comment (jnm2377) [inline src/components/content-switcher/_content-switcher.scss]: same for these

File: src/components/content-switcher/_content-switcher.scss [hunk 3]
```diff
@@ -162,6 +163,16 @@
...
+  }
+
+  .#{$prefix}--content-switcher-btn:last-child {
+    border-top-right-radius: 4px;
+    border-bottom-right-radius: 4px;
```
Comment (jnm2377) [inline src/components/content-switcher/_content-switcher.scss]: and these

**Review by jnm2377 — APPROVED** [review]
Looks good besides the comments I left.

**Comment by lovemecomputer** [comment]
@aledavila good work!! 

❓ @laurenmrice  : just want to check if we want to put any minimum or maximum widths, on either the entire ContentSwitcher wrapper, and/or on individual buttons? (possibly ellipsis overflow as if we're putting a max width)

i think it looks great as-is, but just wanted to point out some oddness that can come with edge cases:

Screen Shot 2019-03-13 at 11 03 22 AM
Screen Shot 2019-03-13 at 11 06 06 AM

**Comment by aledavila** [comment]
true @laurenmrice do we want to add a min or max widths on this?

**Comment by aledavila** [comment]
@lovemecomputer I wonder if not putting constraints on the content switcher will give flexibility to users. Since its on them to not make it overwhelmingly long and doubt they have a sentence in the switcher.  If we don't give them widths then it allows them to set their own depending on how they want their layout to look ?

**Review by lovemecomputer — CHANGES_REQUESTED** [review]
@aledavila we DMed about notes after discussion with @laurenmrice, posting here for tracking:

- [x]  all ContentSwitcher buttons should be same with. 
    - for the time being, we will resolve this by setting the ContentSwitcher to be full width, and each button should be an equal proportion of the width
- [x]  ContentSwitcher button text should do the ellipsis overflow thing instead of wrapping to a next line
- [ ]  currently there is a disparity btw chrome & other browers: chrome always displays the focus outline after clicking, but other browsers on mac only display the outline when using keyboard navigation. we want to make this consistent across all browsers
    - for the time being, we will make sure the border display on safari & firefox after clicking. it's probably the easiest thing to implement, and is more consistent with other components.

other than that i think all looks good!!

**Comment by laurenmrice** [comment]
Thanks Audrey for documenting ! @aledavila let me know when you are ready for another review :)

**Comment by aledavila** [comment]
Note: focus isn't triggered in FF and safari but is consistent with all components. For consistency will leave as is. We can address post-v10 how we want to and if we want to address library wide.

**Review by laurenmrice — CHANGES_REQUESTED** [review]
**Changes:**

- [x] The small rule is showing between hover states. it needs to be hidden or overlapped by the hover.
Screen Shot 2019-03-13 at 4 30 48 PM

- [x] When focusing on a section that is not currently selected. The text color should not change to text-01 it should remain text-02.
Screen Shot 2019-03-13 at 4 33 03 PM

- [x] Could we make the default widths for the sections to be 192px? instead of 265px?

- [x] If the ellipses occurs, we should include the interaction of having the tooltip to show the full text. This will be the same tooltip used for progress indicator.

**Comment by asudoh** [comment]
Thanks @laurenmrice for another round of review? May I ask what are absolute `v10` no-go items in the latest list?

**Comment by asudoh** [comment]
Per out discussion, tracking the remainder at #2092.

**Review by asudoh — APPROVED** [review]
LGTM except the one we tack in #2092 - Thanks @aledavila!

**Pull request merged**